src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -707,3 +733,31 @@ def _get_metadata_counts(
                     df[column_name].isin(nicknames.index).sum()
                 )
     return metadata_df
+
+
+def update_guardian_duplication_row_counts(melted_df: pd.DataFrame) -> pd.DataFrame:
+    # Update guardian duplication row counts to make number of rows match anticipated
```

Comment:
nit: we typically wrap docstrings with """""" and then in-line comments with #. Maybe it's just me, but it makes it easier for me to distinguish between the function's comment and specific step comments.